components/SidebarNavigation/MenuItemButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import styled from "styled-components";
+import { Button as UnstyledButton } from "@components/Button";
+
+type MenuItemProps = {
+  className?: string;
+  text: string;
+  iconSrc: string;
+  onClick: () => void;
+  isCollapsed: boolean;
+};
+
+const ListItem = styled.li`
```

Comment:
Why doesn't this component have a top margin as the one in MenuItemLink?

```
margin-top: ${({ theme }) => theme.spacing[1]};
```

Reply:
Turns out the button is missing some styles. I refactored the button to reuse the styles from the link component.